app/main.js

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
 
 /* components */
 import App from './App.jsx';
 
 /* redux */
 import Store from './reducers/index.js';
 import { createStore } from 'redux';
-const store = createStore(Store);
 
-ReactDOM.render(<App store={store} />, document.getElementById('app'));
+const store = createStore(Store);
 
-store.subscribe(render);
+ReactDOM.render(
+  <Provider store={store}>
```

Comment:
because this is supposed to give all containers access to state...?
But I don't know how to get to the state after this..